package.json

```diff
@@ -11,6 +11,7 @@
     "mousetrap": "1.6.0",
     "ms": "0.7.1",
     "object-values": "1.0.0",
+    "php-escape-shell": "^1.0.0",
```

Comment:
Sorry, i don't understand: is pinned dependency a greenkeeper.io thing or you mean set it with the exact version without the `^`? I apologize for the ignorance about this :)